schunk_gripper_driver/schunk_gripper_driver/tests/test_logging.py

```diff
@@ -38,4 +38,51 @@ def test_driver_doesnt_fill_disk_space_by_default(log_monitor, lifecycle_interfa
         driver.change_state(Transition.TRANSITION_DEACTIVATE)
         driver.change_state(Transition.TRANSITION_CLEANUP)
 
-    # log_monitor does the final asserting
+    log_helper.change_log_level("INFO")
+    # log_level_checker does the final asserting
+
+
+@skip_without_gripper
+def test_driver_rejects_invalid_log_level():
```

Comment:
I think I'm missing a fixture here. What does this test need? `ros2, driver, lifecycle_interface`?